src/main/java/com/matt/forgehax/mods/StepMod.java

```diff
@@ -29,27 +32,89 @@
           .description("how high you can step")
           .defaultTo(1.2f)
           .min(0f)
+          .success(__ -> {
```

Comment:
change this from `.success` to `.changed` you can also remove the brackets for this lambda and do `.success(__ -> MC.addScheduledTask(() -> {...` instead

Reply:
Outdated already